RNCClipboard.podspec

```
@@ -3,17 +3,18 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
 
 Pod::Spec.new do |s|
-  s.name         = "RNCClipboard"
-  s.version      = package['version']
-  s.summary      = package['description']
-  s.license      = package['license']
+  s.name             = "RNCClipboard"
+  s.version          = package['version']
+  s.summary          = package['description']
+  s.license          = package['license']
 
-  s.authors      = package['author']
-  s.homepage     = package['homepage']
-  s.platform     = :ios, "9.0"
+  s.authors          = package['author']
+  s.homepage         = package['homepage']
+  s.platforms        = { :ios => "9.0", :osx => "10.14" }
```

Does this build on 10.13? Which is the deployment target for RN macOS v0.62.x.

```suggestion
  s.platforms        = { :ios => "9.0", :osx => "10.13" }
```

---

The iOS deployment target has also been bumped to `10.0` in v0.63.x btw, but that’s orthogonal to this PR. /cc @Naturalclar

---

How much do we care about 10.13? 11.0 comes out in two weeks and the usual cadence I'm familiar with is to support current version - 2 (so 11.0, 10.14, 10.15)

---

The convention has been to follow the minimum deployment target set by the RN platform that a lib targets. If the lib does things that are really only supported from a newer version onward or much harder to support otherwise, then it’s of course fine to deviate.